test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SWAEndToEndTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.WebHostEndToEnd
+{
+    /// <summary>
+    /// Tests verifying the Static Web Apps deployment configuration.
+    /// </summary>
+    public class SWAEndToEndTests : IClassFixture<SWAEndToEndTests.TestFixture>
+    {
+        private TestFixture _fixture;
+
+        public SWAEndToEndTests(TestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public async Task InvokeFunction_AnonymousLevel_Succeeds()
+        {
+            // send along a code query param, and ensure that the invocation succeeds
+            // the token isn't interpreted as a function key in this case since keys are disabled
+            var content = new StringContent(JsonConvert.SerializeObject(new { scenario = "swa" }));
+            var response = await _fixture.Host.HttpClient.PostAsync("api/HttpTrigger-Scenarios?code=test", content);
+            response.EnsureSuccessStatusCode();
+
+            // TODO: should this verify that the code was interpreted as a parameter?
```

Comment:
I don't think so - if it were interpreted as an auth code, we'd attempt to access secrets and we'd have a 500 right? You could verify that in the debugger by forcing the auth handler down that path, but that's what I saw.

Reply:
That is what happens in that scenario, yes, and it can be assumed that if not interpreted as an auth code, it'll be read as a parameter. I looked back at Anthony's requirements in https://github.com/Azure/azure-functions-host/issues/5922#issuecomment-781760639 and 'parameter value accessible in function' was explicitly called out, so I thought to verify.